pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>plugins</artifactId>
         <groupId>org.igniterealtime.openfire</groupId>
-        <version>4.3.0-beta</version>
+        <version>4.4.0-beta</version>
```

Comment:
Why do we need a newer version of Openfire?

Reply:
.factorypath file removed and version decreased.